pkg/util/runtimeconfig/manager.go

```diff
@@ -44,20 +51,21 @@ type Manager struct {
 	config    interface{}
 
 	configLoadSuccess prometheus.Gauge
+	configHash        *prometheus.GaugeVec
 }
 
 // NewRuntimeConfigManager creates an instance of Manager and starts reload config loop based on config
 func NewRuntimeConfigManager(cfg ManagerConfig, registerer prometheus.Registerer) (*Manager, error) {
 	mgr := Manager{
```

Comment:
As cfg.LoadPath is required now, can we add a check and return error if it's empty?

Reply:
:+1: